Source/JavascriptEditor/JavascriptEditorViewport.cpp

```diff
@@ -70,6 +124,19 @@ class FJavascriptEditorViewportClient : public FEditorViewportClient
 		}
 	}
 
+	virtual void Draw(FViewport* Viewport, FCanvas* Canvas) override
+	{
+		FEditorViewportClient::Draw(Viewport, Canvas);
+
+		if (Widget->SelectionBeginPoint.X >= 0)
+		{
+			FCanvasBoxItem Box(Widget->SelectionBeginPoint, Widget->SelectionEndPoint - Widget->SelectionBeginPoint);
+			Box.SetColor(FColor::White);
+			Box.LineThickness = 1.0;
+			Box.Draw(Canvas);
```

Comment:
This function seems to handle draw behavior for specific use case. I think there should be some functions which exposes FCanvas*Item accessible to BP.